test/test_model.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
Why don't you make this an executable file in Git?

Reply:
Done.